.eslintrc.yaml

```diff
@@ -1,11 +1,25 @@
-extends: '@commercetools-frontend/eslint-config-mc-app'
+extends:
```

Comment:
The eslint config is one of the messy things, probably due the unfortunate combination of our own config, the XO config and typescript.